multi-node/aws/artifacts/scripts/install-controller.sh

```diff
@@ -64,6 +64,13 @@ function init_config {
 	done
 }
 
+function install_kubelet {
+	mkdir /opt
+	cd /opt
+	curl -sLO https://storage.googleapis.com/kubernetes-release/release/v1.1.1/bin/linux/amd64/kubelet
```

Comment:
We should sub `v1.1.1` in this line with ${K8S_VER} so it doesn't need updated with each version change.